core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -232,7 +233,13 @@ import com.datastax.driver.core.utils.UUIDs
               case (_, publishedTag) =>
                 if (publishedTag.equals(session.tag)) {
                   log.debug("[{}] Received pub sub tag update for our tag, initiating query", stageUuid)
-                  continue()
+                  if (settings.eventsByTagEventualConsistency == Duration.Zero) {
+                    continue()
+                  } else if (settings.eventsByTagEventualConsistency < settings.refreshInterval) {
+                    // No point scheduling for EC if the QueryPoll will come in beforehand
```

Comment:
Is this correct?
Let's say resfreshInterval 10.seconds and relying on quick notifications via pub/sub. Then those notifications will be ignored if EC delay is 5.seconds.

Reply:
In that case EC would be < refresInterval so we'd schedule a TagNotification for 5s